argus/config_generator/windows/cb_init.py

```diff
@@ -93,7 +94,11 @@ def _get_service(self, service_type):
             This can be HTTP, ConfigDrive, EC2, OpenNebula,
             CloudStack or MAAS.
         """
-        return '.'.join([util.SERVICES_PREFIX, self.SERVICES[service_type]])
+        if self.SERVICES[service_type]:
```

Comment:
you can remove the return/else and leave only
 if self.SERVICES[service_type]:
return '.'.join([util.SERVICES_PREFIX,
 +                             self.SERVICES[service_type]])